src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2916,19 +2919,29 @@ public ConstructorEdit(INamedTypeSymbol oldType)
 
                         if (editKind == SemanticEditKind.Update)
                         {
+                            AnalyzeCustomAttributes(oldSymbol, newSymbol, capabilities, diagnostics, semanticEdits, syntaxMap, cancellationToken);
+
                             // The only update to the type itself that's supported is an addition or removal of the partial modifier,
                             // which does not have impact on the emitted type metadata.
                             if (newSymbol is INamedTypeSymbol)
                             {
                                 continue;
                             }
 
-                            // The field/property itself is being updated. Currently we do not allow any modifiers or attributes to be updated.
+                            // The field/property itself is being updated. Currently we do not allow any modifiers to be updated. Attribute
+                            // updates will have been handled already
                             if (newSymbol is IFieldSymbol or IPropertySymbol)
                             {
                                 continue;
                             }
 
+                            // The only updates allowed for a parameter or type parameter is an attribute change, but we only need the edit
+                            // for the containing symbol which will be handled elsewhere.
+                            if (newSymbol is IParameterSymbol or ITypeParameterSymbol)
+                            {
+                                continue;
+                            }
+
 #if TODO_ACCESSORS
```

Comment:
>TODO_ACCESSORS [](http://example.com/codeflow?start=4&length=14)

Did I forget to delete this in my PR?

Reply:
Yep. Did you mean to delete it? The linked issue is still unresolved so I left it just in case.